### PR TITLE
add new terms

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
1. MONDO:0800459 noxacusis or pain hyperacusis
2. MONDO:0800469 ACD-related telomere biology disorder
3. 'ACD-related long telomere syndrome'
4. 'ACD-related short telomere syndrome'
5. MONDO:0800485 AKT3-related overgrowth spectrum
6. MONDO:0800487	CTNNB1-related neurodevelopmental disorder and/or vitreoretinopathy

close https://github.com/monarch-initiative/mondo/issues/7643 
close https://github.com/monarch-initiative/mondo/issues/7605 
close https://github.com/monarch-initiative/mondo/issues/7599 
close https://github.com/monarch-initiative/mondo/issues/7572

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -526173,6 +526211,18 @@ is_a: MONDO:0019042 {source="https://clinicalgenome.org/affiliation/40130/"} ! m
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7976 {source="https://clinicalgenome.org/affiliation/40130/"} ! NR2F2
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6666" xsd:anyURI
 
+[Term]
+id: MONDO:0800459
+name: noxacusis or pain hyperacusis
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:526216]: Should we be creating term names that include "or" in the name especially when each side of the "or" statement is also included as synonyms?

Also, is this a disease vs. a phenotype?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -526173,6 +526211,18 @@ is_a: MONDO:0019042 {source="https://clinicalgenome.org/affiliation/40130/"} ! m
...
+def: "Noxacusis/pain hyperacusis is characterized by individuals who experience otalgia or pain (e.g., dull ache, burning, sharp, stabbing pain and throbbing pain) in response to everyday sounds. This differs clinically from those whose primary symptom is the perception of everyday sounds as excessively loud (termed loudness hyperacusis)." [https://liu.med.jhmi.edu/people/chang-liu-ph-d/, https://www.hopkinsmedicine.org/profiles/details/elisabeth-glowatzki, https://www.hopkinsmedicine.org/profiles/details/paul-fuchs, PMID:25104073, PMID:26553995]
+synonym: "auditory nociception" EXACT [https://liu.med.jhmi.edu/people/chang-liu-ph-d/, https://www.hopkinsmedicine.org/profiles/details/elisabeth-glowatzki, https://www.hopkinsmedicine.org/profiles/details/paul-fuchs, PMID:25104073, PMID:26553995]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:526218]: should researcher home page urls be used as a source

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -526263,6 +526313,15 @@ intersection_of: MONDO:0019119 ! muscular channelopathy
...
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6882" xsd:anyURI
 
+[Term]
+id: MONDO:0800469
+name: ACD-related telomere biology disorder
+def: "A complex set of inherited conditions caused by a pathogenic variant(s) in the ACD gene that results in aberrant telomere biology." [https://orcid.org/0009-0007-0138-2054, PMID:20301779]
+synonym: "ACD-related telomeropathy" EXACT [https://orcid.org/0009-0007-0138-2054]
+is_a: MONDO:0100137 {source="https://orcid.org/0009-0007-0138-2054"} ! telomere syndrome
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25070 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7605" xsd:anyURI
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:526323]: looks good

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -519289,6 +519297,34 @@ property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeTrait"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7616" xsd:anyURI
 is_obsolete: true
 
+[Term]
+id: MONDO:0100569
+name: ACD-related short telomere syndrome
+def: "A spectrum of conditions, including dyskeratosis congenita, Hoyeraal-Hreidarsson syndrome, hereditary aplastic anemia, and pulmonary fibrosis, typically characterized by shortened telomeres due to a pathogenic variant(s) in ACD that results in impaired telomere maintenance." [https://orcid.org/0009-0007-0138-2054]
+is_a: MONDO:0800469 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD-related telomere biology disorder
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25070 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:519306]: Is this property still being added for new classes?
- It's not in the documentation for adding a new term https://mondo.readthedocs.io/en/stable/editors-guide/add-new-terms/#add-a-new-term
- It was not added for the new MONDO:0800469, the parent of this term

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -519289,6 +519297,34 @@ property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeTrait"}
...
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7605" xsd:anyURI
+
+[Term]
+id: MONDO:0100570
+name: ACD-related long telomere syndrome
+def: "A telomere biology disorder typically characterized by increased telomere length due to a pathogenic variant in the ACD gene that may cause familial melanoma." [https://orcid.org/0009-0007-0138-2054]
+is_a: MONDO:0800469 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD-related telomere biology disorder
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25070 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:519315]: still used?

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -526383,6 +526442,14 @@ is_a: MONDO:0005217 {source="PMID:29298659", source="https://orcid.org/0009-0007
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9386 {source="PMID:29298659", source="https://orcid.org/0009-0007-0138-2054", source="https://www.clinicalgenome.org/affiliation/40104/"} ! PRKAG2
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6446" xsd:anyURI
 
+[Term]
+id: MONDO:0800485
+name: AKT3-related overgrowth spectrum
+def: "Any overgrowth syndrrome where the cause of the disease is a gain-of-function variant in the AKT3 gene" [PMID:28969385]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:526448]: typo: syndrrome --> syndrome

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -519289,6 +519297,34 @@ property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeTrait"}
...
+name: ACD-related long telomere syndrome
+def: "A telomere biology disorder typically characterized by increased telomere length due to a pathogenic variant in the ACD gene that may cause familial melanoma." [https://orcid.org/0009-0007-0138-2054]
+is_a: MONDO:0800469 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD-related telomere biology disorder
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25070 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7605" xsd:anyURI
+
+[Term]
+id: MONDO:0100571
+name: CTNNB1-related neurodevelopmental disorder and/or vitreoretinopathy
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:519320]: is this one or two diseases? when i search i find these as separate diseases.

**Review by twhetzel — COMMENTED** [review]
a few comments

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:519306]: added via protege

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:519315]: added via protege

File: src/ontology/mondo-edit.obo (hunk 7 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:519320]: for clingen requests, this is okay

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:519320]: but in this case, these parents are not accurate, should be more general

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:519320]: revised parent

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:526216]: revise label

**Comment by nicolevasilevsky** [comment]
@twhetzel this is ready for re-review/merging

**Review by twhetzel — APPROVED** [review]
looks great!

**Pull request merged**